src/game_battlealgorithm.cpp

```diff
@@ -676,7 +676,7 @@ void Game_BattleAlgorithm::AlgorithmBase::Apply() {
 	if (GetAffectedDefense() != -1) {
 		int def = GetAffectedDefense();
 		target->ChangeDefModifier(IsPositive() ? def : -def);
-		if (IsAbsorb()) {
+		if (IsAbsorb() && !IsPositive() && Player::IsRPG2k3()) {
```

Comment:
Confirmed that stats cannot be absorbed in 2k. I checked with RE and tested it in battle.